mods/cnc/chrome/mapchooser.yaml

```diff
@@ -14,62 +14,75 @@ Container@MAPCHOOSER_PANEL:
 			Text: Select Map
 		Background@bg:
 			Width: PARENT_RIGHT
-			Height: 540
+			Height: 570
 			Background: panel-black
 			Children:
+				Label@ORDERBY_LABEL:
+					X: PARENT_RIGHT - 271
+					Y: 14
+					Width: 30
+					Height: 31
+					Font: Bold
+					Align: Center
```

Comment:
`Align: Right` here as well. This is so that if the text is changed (or translated) we won't need to resize and  reposition the widget.